cob_hardware_config/cob4-2/config/docking_linear_nav.yaml

```diff
@@ -16,7 +16,7 @@ goal_threshold_rot: 0.08 #0.08
 speed_threshold_rot: 0.01 #0.08
 
 #tf parameters
-global_frame: station_reflector_link #/map
+global_frame: odom_combined #/map
```

Comment:
I just struggled over this because station_reflector_link does __not__ even exist! I changed it to a frame which is available on all robots even without running navigation. In my tests there was no difference in setting this frame to ```base_link```, ```odom_combined``` or ```abs_xxx_123```. @ipa-mig: Do you know when this frame really makes a difference?